services/integration-content-repository/k8s/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
```

Comment:
apiversion - explains the level of stability and support, since ICR is not working but IAM('v1beta1') does, I suggest changing it to "v1alpha1"